scripts/preprocess_price_prediction_model.py

```diff
@@ -49,21 +42,28 @@ def drop_columns(df):
                        'body_type', 'transmission', 'fuel_type', 'doors', 'seats', 'number_of_owners', 'emission_scheme',
                        'vehicle_location_latitude', 'vehicle_location_longitude', 'imported',
                        'number_of_photos', 'tax', 'cylinders', 'valves', 'combined', 'insurance_group', 'boot_space_seats_down']
-
     return df.drop(columns_to_drop, axis=1)
 
+def combine_CO2_columns(df: pd.DataFrame) -> pd.DataFrame:
+    '''
+    Combine CO2 columns into one.
+    '''
+    co2_emissions = df[(df.co2_emissions.isnull()) & (~df.co2_emission.isnull())].co2_emission
+    
+    for index, value in co2_emissions.items():
+        df.loc[index, 'co2_emissions'] = value
+
+    return df.drop('co2_emission', axis=1)
+
```

Comment:
Was this copied from the cleaning file because you couldn't import it?

Reply:
I did it cause it was only two functions and I the purpose of the preprocessing script is to have all the preprocessing/cleaning functions in them. I wanted it to be self contained. There's still more changes to make on the preprocessing script I think. I won't have to make changes to the `data_cleaning.py` file from now on.

Reply:
ahh i see, makes sense